Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -656,7 +667,8 @@ public Apis.Bigquery.v2.Data.Job DoLoad()
                                 IgnoreUnknownValues = AllowUnknownFields,
                                 AllowJaggedRows = AllowJaggedRows,
                                 AllowQuotedNewlines = AllowQuotedNewlines,
-                                WriteDisposition = WriteMode.ToString()
+                                WriteDisposition = WriteMode.ToString(),
+                                MaxBadRecords = MaxBadRecords ?? 0
```

Comment:
Yes, it is the default API value. https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.load